config/secure-headers.php

```diff
@@ -629,7 +629,18 @@
 
 		// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/object-src
 		'object-src' => [
-			'none' => true,
+			'self' => true, // needed to display embeded pdf files
+			'allow' => array_merge(
+				[
+					'blob:', // required for "live" photos
+				],
```

Comment:
Is this one needed for objects?

Reply:
Probably not, but I wanted to uniformize with media.